website/source/docs/external/lxc.html.md

```diff
@@ -0,0 +1,152 @@
+---
```

Comment:
Shouldn't this be under docs/drivers/external?

Reply:
You're right. Fix has been implemented.